README.adoc

```diff
@@ -144,58 +142,44 @@ Create the `InventoryManager` class in the
 include::finish/src/main/java/io/openliberty/guides/inventory/InventoryManager.java[tags=InventoryManager]
 ----
 
-Notice the metrics annotations added to this class:
-
-The `@Timed` annotation applies to the `get()` method to track how frequently the method is
+Apply the `@Timed` annotation to the `get()` method to track how frequently the method is
 invoked and also how long it takes for the invocation to complete. This annotation has these
 metadata fields:
 
 |===
-|`name` | Optional and sets the name of the metric
-|`description` | Optional and describes the purpose of the metric annotation
+|`name` | Optional; Use this to name of the metric
+|`description` | Optional; Use this to describe the purpose of the metric
 |===
 
-The `@Counted` annotation applies to the `list()` method. This annotation counts how many times you request the list of
-systems methods when you visit the `http://localhost:9080/inventory/systems` URL. This URL retrieves the information for a
-list of all previously registered hosts. See the additional metadata fields:
+Apply the `@Counted` annotation to the `list()` method to count how many times the
+`http://localhost:9080/inventory/systems` URL is accessed. Note the additional metadata field:
 
 |===
-| `absolute` | if `true`, it names the metric the name of the method. If `false`, it attaches the package name and class name
-before the method name
-| `monotonic` | if `true`, the counter counts the number of invocations of the annotated method. If `false`, the counter counts the
-number of currently executing invocations of the annotated method. In other words, it makes the counter increment
-when the annotated method is called and decrement when it returns.
+| `monotonic` | Set to `true` to count the total number of invocations of the annotated method i.e.
+the total number of accesses.
```

Comment:
Suggested changes:
"Set this annotation to `true` to count the total number of invocations of the annotated method, such as the total number of times that the page is accessed."